cli/src/utils/utils.ts

```diff
@@ -207,20 +211,48 @@ export const setPayer = (key: string) => {
   setConfig({ payer: key });
 };
 
-export const getConfig = (): Config => {
+export const getConfig = (filePath?: string): Config => {
+  if (!filePath) filePath = process.env.HOME + CONFIG_PATH + CONFIG_FILE_NAME;
+
   try {
-    const data = fs.readFileSync("config.json", "utf-8");
+    const data = fs.readFileSync(filePath, "utf-8");
     return JSON.parse(data);
   } catch (error) {
     throw new Error("Failed to read configuration file");
   }
 };
 
-export const setConfig = (config: Partial<Config>): void => {
+export function ensureDirectoryExists(dirPath: string): void {
+  if (!fs.existsSync(dirPath)) {
+    fs.mkdirSync(dirPath, { recursive: true });
+  }
+}
+
+export const setConfig = (config: Partial<Config>, filePath?: string): void => {
+  if (!filePath) filePath = process.env.HOME + CONFIG_PATH + CONFIG_FILE_NAME;
```

Comment:
What do you think about handling a custom environment variable first? I mean, having this priority list when looking up for config:

* A path set in `LIGHT_PROTOCOL_CONFIG` environment variable
* `~/.config/light/config.json`

The one specified in the env variable would have a precedence.